metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-hiera",
-  "version": "5.0.2-rc0",
+  "version": "6.0.0",
```

Comment:
Before we do the release, can you also remove the EoL operating systems listed above, in another PR?

Reply:
@bastelfreak yes PR  #335 about this

Reply:
CentOS 8 is EoL as well. Can you please also drop it?